core/vcpu.c

```diff
@@ -2575,10 +2589,31 @@ static void handle_cpuid_virtual(struct vcpu_t *vcpu, uint32_t a, uint32_t c)
              * that is an old i7 system, so the emulator can still utilize the
              * enough extended features of the hardware, but doesn't crash.
              */
-            hw_family = ((state->_eax >> 16) & 0xFF0) |
-                        ((state->_eax >> 8) & 0xF);
-            hw_model = ((state->_eax >> 12) & 0xF0) |
-                       ((state->_eax >> 4) & 0xF);
+             union cpuid_1_eax {
+                 uint32_t raw;
+                 struct {
+                     unsigned steppingID    : 4;
+                     unsigned model         : 4;
+                     unsigned familyID      : 4;
+                     unsigned processorType : 2;
+                     unsigned reserved      : 2;
+                     unsigned extModelID    : 4;
+                     unsigned extFamilyID   : 8;
+                     unsigned reserved2     : 4;
+                 };
+             } cpuid_eax, cpuid_eax2;
```

Comment:
The variable `cpuid_eax2` is not used anywhere. Due to the strict compiler flags in the Darwin builds this triggers an error:
https://travis-ci.com/intel/haxm/jobs/203067260#L592

Reply:
Thanks! Yes, we have more strict compiler flags enabled on Mac. Unused variables should be removed.